resources/translations/fi.edn

```diff
@@ -137,7 +137,7 @@
                           :submitted "%1 lähetti hakemuksen käsiteltäväksi."
                           :unknown "Tuntematon"}
                  :id "Tunniste"
-                 :intro [:div.intro [:p "Täyttämällä ja lähettämällä tämän hakemuksen kaikki hakijat voivat saada oikeuden haettuihin resursseihin. Kunkin hakijan on lisäksi hyväksyttävä käyttöehdot viimeistään hakemuksen käsittelyn jälkeen."]]
+                 :intro [:div.intro [:p "Täytä hakemus ja lähetä se käsiteltäväksi. Hakemukseen voi kutsua jäseniksi useampia hakijoita, ja jokaisen on hyväksyttävä käyttöehdot viimeistään kun hakemus on hyväksytty."]]
```

Comment:
this is missing the point from the english text that access is only granted after accepting the terms of use... but I guess that's fine

Reply:
I thought about that and came to the conclusion that accepting the terms is implied because an application can't be accepted otherwise.